esp/src/src/DiskUsage.ts

```diff
@@ -65,44 +75,88 @@ export class Summary extends FlexGrid {
                 .text(nlsHPCC.loadingMessage)
                 ;
         }
-        this._connection.GetTargetClusterUsageEx(undefined, bypassCachedResult).then(response => {
+        connection.GetTargetClusterUsageEx(this._targetCluster !== undefined ? [this._targetCluster] : undefined, bypassCachedResult).then(response => {
             this._loadingMsg && this._loadingMsg
                 .html('<i class="fa fa-database"></i>')
                 ;
-            response.forEach(details => {
-                if (!this._usage[details.Name]) {
-                    this._usage[details.Name] = {
-                        details,
-                        gauge: new Gauge()
-                            .title(details.Name)
-                            .showTick(true)
-                            .on("click", (gauge: Gauge) => {
-                                this.click(gauge, details);
-                            })
-                    };
-                }
-                this._usage[details.Name].gauge
-                    .value((details.max || 0) / 100)
-                    .valueDescription(nlsHPCC.Max)
-                    .tickValue((details.mean || 0) / 100)
-                    .tickValueDescription(nlsHPCC.Mean)
-                    .tooltip(details.ComponentUsagesDescription)
-                    ;
-            });
+            if (!this._targetCluster) {
+                response.forEach(details => {
+                    if (!this._usage[details.Name]) {
+                        this._usage[details.Name] = {
+                            details,
+                            gauge: new Gauge()
+                                .title(details.Name)
+                                .showTick(true)
+                                .on("click", (gauge: Gauge) => {
+                                    this.click(gauge, details);
+                                })
+                        };
+                    }
+                    this._usage[details.Name].gauge
+                        .value((details.max || 0) / 100)
+                        .valueDescription(nlsHPCC.Max)
+                        .tickValue((details.mean || 0) / 100)
+                        .tickValueDescription(nlsHPCC.Mean)
+                        .tooltip(details.ComponentUsagesDescription)
```

Comment:
No need for translation in tooltips?

Reply:
No - the tooltip is basically info sent back from the server.